src/style.css

```diff
@@ -90,7 +110,15 @@ input[type="submit"]:hover {
 .scoreField {
   width: 200px;
   padding: 0.5rem;
-  border: 3px solid;
+  border: 0;
   font-size: 18px;
   border-radius: 3px;
+  color: purple;
+  font-weight: bolder;
+}
+
+.scoreField::placeholder {
+  color: purple;
```

Comment:
✅ App is properly styled and its layout looks like the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png) 👍🏻